ecosystem/sep-0028.md

```diff
@@ -0,0 +1,37 @@
+## Preamble
+
+```
+SEP: 28
+Title: Ease burden on new users
+Author: @buhrmi
+Track: Informal
+Status: Draft
+Created: 2019-11-26
+Discussion: (see mailing list, waiting for approval)
+```
+
+## Simple Summary
+Remove the `createAccount` transaction, and trustlines to make Steller more accessible to new users.
+
+## Motivation
+The current solution to depositing an asset in a new users account is as follows
+
+1. Create a stellar account in their wallet
+2. Somehow acquire lumens to fund and open the account
+3. Open a trustline for the asset
+4. Provider sends payment
+
+This has a few points of failure, and the burden of asking clients to acquire lumens is too high.
+
+## Abstract
+The proposed solution is to completely remove the `createAccount` transaction and trustlines.
+Any account will be able to hold and unlimited number of assets issued by somebody else, without first creating a trustline.
+
+## Specification
+TODO
+
+## Design Rationale
+TODO
```

Comment:
I think to merge this draft the document needs to go into details about how the removal of create account will take place.